CHANGELOG.md

```diff
@@ -1,11 +1,30 @@
 # Changelog
 
+# 0.9.0 / 2022-10-04
+
+- [FEATURE] New ListVariable plugin for StaticListVariables #547
+- [FEATURE] New Prometheus plugin for ListVariable (LabelName, LabelValues) #565
+- [FEATURE] Variable chaining support for Prometheus variables #579
+- [FEATURE] New editable Markdown panel #553, #555, #563
+- [FEATURE] New datasource spec implemented #554, #577
+- [ENHANCEMENT] Monorepo / build tooling improvements #550, #567, #578, #585, #586
```

Comment:
I would also add here #583.
#578 seems a bit odd comparing to the others PRs because others are really focus on improving the CI.

Reply:
I fixed it!